tuxedo-template-runtime/src/kitties.rs

```diff
@@ -1,126 +1,544 @@
-//! An NFT game inspired by cryptokitties.
```

Comment:
A few sentences about the general flow of the kitty breeding would be helpful. For example, are kitties inherently gendered or can the same kitty act as mom sometimes and dad other times. Maybe explain the tired period, etc.